packages/cody/src/lib/hooks/utils/ui/get-ui-metadata.ts

```diff
@@ -5,7 +5,7 @@ import {parseJsonMetadata} from "@proophboard/cody-utils";
 
 export interface DynamicBreadcrumbMetadata {
   data: string;
-  value: Rule[] | string;
+  label: Rule[] | string;
```

Comment:
That's a small correction of breadcrumb config so that the wording is aligned with the one used in data select.